src/openforms/js/components/admin/tables/ChangelistTable.js

```diff
@@ -1,10 +1,60 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 import ChangelistColumn from './ChangelistColumn';
 import {IconYes, IconNo, IconUnknown} from '../BooleanIcons';
 
 
+const ChangelistTableWrapper = ({ headColumns, children: body, extraModifiers=[] }) => (
+    <div className={classNames('changelist', 'changelist--react', ...extraModifiers.map(mod => `changelist--${mod}`))}>
```

Comment:
May be nice to have a utils function `getBEMClassName` like in the sdk?

Reply:
yeah the thought has occurred... we actually have bem.js as dependency already so I'll see if I can make that do what I want it to do.